### PR TITLE
https://issues.redhat.com/browse/ACM-12115 tech debt

### PR DESCRIPTION
**Pull request by oafischer**
2.11 only

### CODE REVIEW

File: clusters/cluster_lifecycle/clusterclaims.adoc [hunk 1]
```diff
@@ -91,3 +91,20 @@ status:
   version:
     kubernetes: v1.18.3+6c42de8
 ----
+
+[#create-custom-clusterclaims]
+== Create custom ClusterClaims
+
+You can create ClusterClaims with custom names on a managed cluster, which makes it easier to identify them. The custom ClusterClaims are synchronized with the status of the `ManagedCluster` resource on the hub cluster. The following content shows an example of a definition for a customized `ClusterClaim`:
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/clusterclaims.adoc]:
```suggestion
You can create `ClusterClaims` resources with custom names on a managed cluster, which makes it easier to identify them. The custom `ClusterClaims` resource are synchronized with the status of the `ManagedCluster` resource on the hub cluster. The following content shows an example of a definition for a customized `ClusterClaim` resource:
```

File: clusters/cluster_lifecycle/clusterclaims.adoc [hunk 2]
```diff
@@ -91,3 +91,20 @@ status:
...
+----
+apiVersion: cluster.open-cluster-management.io/v1alpha1
+kind: ClusterClaim
+metadata:
+  name: <custom_claim_name>
+spec:
+  value: <custom_claim_value>
+----
+
+The length of `spec.value` field must be 1024 or lower. The `create` permission on resource `clusterclaims.cluster.open-cluster-management.io` is required to create a ClusterClaim.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/clusterclaims.adoc]:
```suggestion
The length of `spec.value` field must be 1024 or lower. The `create` permission on resource `clusterclaims.cluster.open-cluster-management.io` is required to create a `ClusterClaim` resource.
```

File: clusters/cluster_lifecycle/config_manifest_create.adoc [hunk 3]
```diff
@@ -3,13 +3,20 @@
 
 You can configure additional Kubernetes resource manifests during the installation process of creating your cluster. This can help if you need to configure additional manifests for scenarios such as configuring networking or setting up a load balancer. 
 
-Before you create your cluster, you need to add a reference to the `ClusterDeployment` resource that specifies a `ConfigMap` that contains the additional resource manifests. 
+[#config-manifest-create-prereq]
+== Prerequisite
 
-*Note:* The `ClusterDeployment` resource and the `ConfigMap` must be in the same namespace. The following examples show how your content might look. 
+Add a reference to the `ClusterDeployment` resource that specifies a `ConfigMap` that contains the additional resource manifests.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/config_manifest_create.adoc]:
```suggestion
Add a reference to the `ClusterDeployment` resource that specifies a `ConfigMap` resource that contains the additional resource manifests.
```

or 

```suggestion
Add a reference to the `ClusterDeployment` resource that specifies a config map that contains the additional resource manifests.
```

File: clusters/cluster_lifecycle/config_manifest_create.adoc [hunk 4]
```diff
@@ -3,13 +3,20 @@
...
+
+*Note:* The `ClusterDeployment` resource and the `ConfigMap` must be in the same namespace.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/config_manifest_create.adoc]: similar suggestion

File: clusters/cluster_lifecycle/config_manifest_create.adoc [hunk 5]
```diff
@@ -3,13 +3,20 @@
...
+
+[#config-manifest-create-steps]
+== Configuring additional manifests during cluster creation by using examples
+
+If you want to configure additional manifests by using a `ConfigMap` with resource manifests, complete the following steps:
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/config_manifest_create.adoc]: similar suggestion, I'm more inclined to suggest: "config map"

File: clusters/cluster_lifecycle/config_manifest_create.adoc [hunk 6]
```diff
@@ -3,13 +3,20 @@
...
+
+. Create a YAML file and add the following example content.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/config_manifest_create.adoc]:
```suggestion
. Create a YAML file and add the following example content:
```

File: clusters/cluster_lifecycle/config_manifest_create.adoc [hunk 7]
```diff
@@ -37,10 +44,20 @@ data:
       cache_url: "http://192.168.111.1/images"
       rhcos_image_url: "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.201911192044.0/x86_64/rhcos-43.81.201911192044.0-openstack.x86_64.qcow2.gz"
 ----
+*Note:* The example `ConfigMap` contains a manifest with another `ConfigMap` resource. The resource manifest `ConfigMap` can contain multiple keys with resource configurations added in a `data.<resource_name>\.yaml` pattern.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/config_manifest_create.adoc]:
```suggestion
*Note:* The example `ConfigMap` contains a manifest with another `ConfigMap` resource. The resource manifest `ConfigMap` can contain multiple keys with resource configurations added in the following pattern, `data.<resource_name>\.yaml`.
```

File: clusters/cluster_lifecycle/config_manifest_create.adoc [hunk 8]
```diff
@@ -37,10 +44,20 @@ data:
...
 
-* ClusterDeployment with resource manifest `ConfigMap` referenced
 +
-The resource manifest `ConfigMap` is referenced under `spec.provisioning.manifestsConfigMapRef`.
+[source,bash]
+----
+oc apply -f <filename>.yaml
+----
+
+If you want to configure additional manifests by using a `ClusterDeployment` by referencing a resource manifest `ConfigMap`, complete the following steps:
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/config_manifest_create.adoc]: to fix ordered list
Screenshot 2024-07-05 at 11 35 14 AM
```suggestion
+
If you want to configure additional manifests by using a `ClusterDeployment` by referencing a resource manifest `ConfigMap`, complete the following steps:
```

File: clusters/cluster_lifecycle/release_images_specify.adoc [hunk 9]
```diff
@@ -172,4 +172,4 @@ The creation process uses the merged release images to create the cluster.
 
 - See the link:https://github.com/stolostron/acm-hive-openshift-releases[acm-hive-openshift-releases] GitHub repository for the YAML files that reference the release images.
 
-- See the link:https://github.com/stolostron/cluster-image-set-controller/blob/main/README.md[cluster-image-set-controller GitHub repository] GitHub repository to learn how to enable enable automatic updates of `ClusterImageSets` from a forked GitHub repository.
\ No newline at end of file
+- See the link:https://github.com/stolostron/cluster-image-set-controller/blob/main/README.md[cluster-image-set-controller GitHub repository] GitHub repository to learn how to enable enable automatic updates of `ClusterImageSets` from a forked GitHub repository.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/release_images_specify.adoc]:
```suggestion
- See the link:https://github.com/stolostron/cluster-image-set-controller/blob/main/README.md[cluster-image-set-controller GitHub repository] to learn how to enable enable automatic updates of `ClusterImageSets` resources from a forked GitHub repository.
```

File: clusters/cluster_lifecycle/upgrade_cluster.adoc [hunk 10]
```diff
@@ -5,23 +5,29 @@ After you create {ocp} clusters that you want to manage with {mce-short}, you ca
...
 
 *Important:* You cannot upgrade Red Hat OpenShift Kubernetes Service managed clusters or {ocp-short} managed clusters on Red Hat OpenShift Dedicated by using the {mce-short} console. 
 
+[#upgrading-your-cluster-conn-steps]
+== Upgrading your cluster in a connected environment
+
 To upgrade your cluster in a connected environment, complete the following steps:
 
-. From the navigation menu, navigate to *Infrastructure* > *Clusters*. If an upgrade is available, it is shown in the _Distribution version_ column.
+. From the navigation menu, navigate to *Infrastructure* > *Clusters*. If an upgrade is available, it appears in the _Distribution version_ column.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/upgrade_cluster.adoc]:
```suggestion
. From the navigation menu, go to *Infrastructure* > *Clusters*. If an upgrade is available, it appears in the _Distribution version_ column.
```

File: clusters/cluster_lifecycle/upgrade_cluster.adoc [hunk 11]
```diff
@@ -37,7 +43,7 @@ To select a channel for your cluster, complete the following steps:
 
 . Select the name of the cluster that you want to change to view the _Cluster details_ page. If a different channel is available for the cluster, an edit icon is displayed in the _Channel_ field.
 
-. Click the edit icon to modify the setting in the field. 
+. Click the edit icon to change the setting in the field.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/upgrade_cluster.adoc]:
```suggestion
. Click the *Edit* icon to change the setting in the field.
```

File: clusters/cluster_lifecycle/upgrade_cluster_disconnected.adoc [hunk 12]
```diff
@@ -1,11 +1,11 @@
 [#upgrading-disconnected-cluster]
 = Upgrading a disconnected cluster
 
-You can use {cincinnati} with {mce-short} to upgrade cluster in a disconnected environment.
+You can use {cincinnati} with {mce-short} to upgrade clusters in a disconnected environment.
 
 In some cases, security concerns prevent clusters from being connected directly to the internet. This makes it difficult to know when upgrades are available, and how to process those upgrades. Configuring {cincinnati-short} can help. 
 
-{cincinnati-short} is a separate operator and operand that monitors the available versions of your managed clusters in a disconnected environment, and makes them available for upgrading your clusters in a disconnected environment. After {cincinnati-short} is configured, it can perform the following actions:
+{cincinnati-short} is a separate operator and operand that monitors the available versions of your managed clusters in a disconnected environment, and makes them available for upgrading your clusters in a disconnected environment. After you configure {cincinnati-short}, it can perform the following actions:
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/upgrade_cluster_disconnected.adoc:8]: Broken link in this file 
Screenshot 2024-07-05 at 11 44 15 AM

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: clusters/cluster_lifecycle/config_manifest_create.adoc (hunk 3 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/config_manifest_create.adoc]: We seem to use config map a lot in other parts of the doc, I'll go with your second suggestion

File: clusters/cluster_lifecycle/upgrade_cluster_disconnected.adoc (hunk 12 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/upgrade_cluster_disconnected.adoc:8]: I think I was able to fix it, looks like the file moved

**Review by dockerymick — APPROVED** [review]
Thanks Oliver!

**Pull request merged**